detection/rgbd_detectors/rwth_upper_body_detector/src/AncillaryMethods.cpp

```diff
@@ -395,7 +393,7 @@ Vector<double> AncillaryMethods::conv1D(const Vector<double> &vec, const Vector<
         }
     }
 
-    return result;
+    vec = result;
```

Comment:
Are you sure this is correct? vec is declared as constant (how can this even compile? doesn't the assignment operator require a non-const lvalue?)